code/datums/materials/basemats.dm

```diff
@@ -247,3 +247,22 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	. = ..()
 	if(istype(source, /obj/item))
 		qdel(source.GetComponent(/datum/component/fantasy))
+
+/datum/material/hot_ice
+	name = "hot ice"
+	id = "hot ice"
+	desc = "A weird kind of ice, feels warm to the touch"
+	color = "#88cdf1"
+	alpha = 150
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/hot_ice
+	value_per_unit = 0.5
+	beauty_modifier = 0.5
+
+/datum/material/hot_ice/on_applied(atom/source, amount, material_flags)
+	. = ..()
+	source.AddComponent(/datum/component/hot_ice, "plasma", amount*500, amount*20+300)
+
+/datum/material/hot_ice/on_removed_obj(atom/source, amount, material_flags)
```

Comment:
any reason this isn't on_removed?

Reply:
nope, just an error while copying stuff around, fixing it in the next commit